### PR TITLE
Python Program that fetches top trending news

### PR DESCRIPTION
**Pull request by SKAUL05**
Python script that uses a news API to fetch top trending news and then prints it.

### CODE REVIEW

File: web_programming/fetch_news.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created by sarathkaul on 12/11/19
+
+import requests
+
+_NEWS_API = "https://newsapi.org/v1/articles?source=bbc-news&sortBy=top&apiKey=4dbc17e007ab436fb66416009dfb59a8"
```
Comment (cclauss) [inline web_programming/fetch_news.py]: Do you ___really___ want to publish ___your___ apiKey?!?
```python
_NEWS_API = "https://newsapi.org/v1/articles?source=bbc-news&sortBy=top&apiKey="
```

File: web_programming/fetch_news.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+def fetch_bbc_news():
```
Comment (cclauss) [inline web_programming/fetch_news.py]:
```python
def fetch_bbc_news(bbc_news_api_key: str) -> None:
    # fetching a list of articles in json format
    bbc_news_page = requests.get(_NEWS_API + bbc_news_api_key).json()
    # each article in the list is a dict
    for i, article in enumerate(bbc_news_page["articles"], 1):
        print(f"{i}.) {article['title']}")
```
https://docs.python.org/3/library/functions.html#enumerate

File: web_programming/fetch_news.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    for a_article in article:
+        results.append(a_article)
+
+    for a_result in range(len(results)):
+        # printing all trending news
+        print(str(a_result + 1) + ".) ", results[a_result]['title'])
+
+
+if __name__ == '__main__':
+    fetch_bbc_news()
```
Comment (cclauss) [inline web_programming/fetch_news.py]:
```python
    fetch_bbc_news(bbc_news_api_key="<Your BBC News API key goes here>")
```

**Comment by SKAUL05** [comment]
> Nice! Please change the filename to `fetch_bbc_news.py` to leave room to add other news feeds in the future.

Changes Done

**Pull request merged**